README.rst

```diff
@@ -39,8 +39,7 @@ Current features
 * Easy long-term sequence data deposition to the European Nucleotide Archive (ENA),
   part of the European Bioinformatics Institute (EBI) for private and public
   studies.
-* Raw data processing for `Target Gene, Metagenomic, Metabolomic and BIOM files <https://qiita.ucsd.edu/static/doc/html/processingdata/index.html#processing-recommendations>`. BIOM files can be added as new preparation files for downstream analyses; however, this cannot be made public.
-
+* Raw data processing for `Target Gene, Metagenomic, Metabolomic, Genome Isolates and BIOM files <https://qiita.ucsd.edu/static/doc/html/processingdata/index.html#processing-recommendations>`__. NOTE: BIOM files can be added as new preparation files for downstream analyses; however, this cannot be made public in the system.
 * Basic downstream analyses using Qiime2.
```

Comment:
I think highlighting that processed qiita files can also be easily integrated to  stand-alone QIIME 2 in case the user wants to use 3rd party plugins not available in qiita, how about we include the link to the qiita-to-QIIME 2 tutorial for this? https://forum.qiime2.org/t/transferring-qiita-artifacts-to-qiime2/4790

Reply:
Good point, I'll add links to that information in the Qiita help and on how to download files.